dockerhub/teamcity-server/README.md

```diff
@@ -44,7 +48,35 @@ To achieve that, you can pass an additional
 started with an alternative `server-https-proxy.xml` configuration file which enables HTTPS options.
 
 Alternatively, you can use a custom Tomcat configuration (see below).
-                       
+
+
+#### Configuring HTTPS Access to TeamCity Server
+
+If a TeamCity Server uses the [HTTPS connection](https://www.jetbrains.com/help/teamcity/using-https-to-access-teamcity-server.html), it transmits encrypted traffic through port `443` by default. 
```

Comment:
This is not only about TeamCity, but any web server. I'm not sure we need to describe it here.

Reply:
I don't get the issue :) To me it's a fine logical chain: "you can set up https -> this will require using port 443 -> ports under 1000 (including this 443) are privileged and can be unaccessible -> solution"

Reply:
OK, we can leave it :)